File: Java/프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.java
import java.util.*;

class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        int max1=0, max2=0, min1=0, min2=0;
        Arrays.sort(numbers);
        
        for(int i=0; i<numbers.length; i++) {
                max1 = numbers[numbers.length-1];
                max2 = numbers[numbers.length-2];
                min1 = numbers[0];
                min2 = numbers[1];
        }
        if(min1*min2 > max1 * max2)
            answer = min1*min2;
        
        else
            answer = max1 * max2;
        
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.js
function solution(numbers) {

    let a=0, b=0;
    numbers.sort((a, b) => a-b);
    a = numbers[0] * numbers[1];
    b = numbers[numbers.length-2] * numbers[numbers.length - 1];
    return Math.max(a, b);
}